manifests/charts/default/templates/mutatingwebhook.yaml

```diff
@@ -38,6 +38,8 @@ metadata:
     operator.istio.io/component: "Pilot"
     app: sidecar-injector
     release: {{ .Release.Name }}
+    app.kubernetes.io/name: "sidecar-injector"
```

Comment:
This is not really a component anymore. Do we want to use the legacy name to align with the `app` in this object, or change it to `istiod` to align with other objects?

Reply:
+1 on just using `istiod` for this.